tests/docker/run-tests.sh

```
@@ -94,38 +95,35 @@ _EOF_
-	JOIN spock.node n
-	ON e.remote_origin = n.node_id
-	WHERE e.operation = 'UPDATE'
-	AND n.node_name = 'n1'
-	AND e.remote_new_tup::text LIKE '%\"trigger missing key on UPDATE\"%';
-	")
-
-  if [ "$elog_entries" -ne 1 ];
+  # spock.missing_update_to_insert is on by default, so the UPDATE of the
+  # missing row is applied as an INSERT rather than logged as an exception.
+  echo "Checking the converted UPDATE now..."
+  converted_row=$(psql -A -t -h ${peer_names[0]} -c \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)